src/components/branding/style.module.css

```diff
@@ -1,5 +1,3 @@
-@import '~style/helpers';
```

Comment:
Interestingly, this change shaves a huge chunk off our CSS bundles -- every import of `helpers.less` here (which in turn imported the `variables.less`) resulting in all of our `:root { }` CSS vars ending up in the bundle.

Loading the tutorial, for example, loads the page with 4 copies of our CSS vars. Yikes.

Reply:
Wow, reduces transfer size by 4.5kb and total size by nearly 7kb.

The page is still 381kb transferred so it's not a measurable improvement of course, just nice to see.